src/ekomark/benchmark/runner.py

```diff
@@ -248,7 +249,13 @@ def log(self, theory, _, pdf, me, ext):
         for q2 in q2s:
             log_tab = dfdict.DFdict()
             ref_pdfs = ext["values"][q2]
-            res = pdf_grid[q2]
+            # find the first hit, regardless of the flavor, since others can not deal with them appearing in parallel
+            eps = [ep for ep in pdf_grid.keys() if np.isclose(ep[0], q2)]
+            if len(eps) != 1:
+                if len(eps) < 1:
```

Comment:
```suggestion
                if len(eps) == 0:
```
As specific as possible.